src/main/java/com/mraof/minestuck/data/GeneratedGristCostConfigProvider.java

```diff
@@ -41,6 +41,7 @@ protected void addEntries()
 		serializer(RecipeSerializer.SHAPED_RECIPE);
 		serializer(RecipeSerializer.SHAPELESS_RECIPE);
 		serializer(MSRecipeTypes.NON_MIRRORED);
+		type(RecipeType.SMITHING, SmithingInterpreter.INSTANCE);
```

Comment:
Now that you've merged changes, this line has become a duplicate. Please remove it and rerun "runData" to also remove changes to "grist_cost_geneeration_recipes".

Reply:
The changes to data is still there. You have not yet run "runData" and committed the data file changes. If you've run "runData", but the generated files aren't updating as they should, it might be that the datagen cache doesn't match the actual generated files. To clear the cache, you can just delete the cache folder in the "generated" folder.